constraints.txt

```diff
@@ -49,25 +49,25 @@ zope.cachedescriptors==5.0
 zope.component==6.0
 zope.configuration==5.0.1
 zope.container==6.0
-zope.contentprovider==5.0
+zope.contentprovider==6.0
 zope.contenttype==5.1
 zope.datetime==5.0.0
 zope.deferredimport==5.0
 zope.deprecation==5.0
 zope.dottedname==6.0
 zope.event==5.0
-zope.exceptions==5.0.1
+zope.exceptions==5.1
 zope.filerepresentation==6.0
 zope.globalrequest==2.0
 zope.hookable==6.0
 zope.i18n==5.1
 zope.i18nmessageid==6.1.0
-zope.interface==6.4.post2
+zope.interface==7.0.1
```

Comment:
Should we wait with the use of `zope.interface>=7.0`? "https://github.com/zopefoundation/zope.interface/issues/315" reports a segmentation fault (for the use of `zope.interface.7.0.1` in `Django`.

Reply:
Good catch. That error report sounds seriously strange and it's unclear if this is Django-specific, so out of an abundance of caution I reverted that pin.